apps/kyberswap-interface/package.json

```diff
@@ -171,7 +171,7 @@
     "use-sound": "^5.0.0",
     "util": "^0.12.5",
     "uuid": "^11.1.0",
-    "viem": "^2.40.3",
+    "viem": "^2.43.1",
```

Comment:
@linhbkhn95 why you also upgrade viem?

Reply:
my mac got issue with old version. Then, i have to update new version to solve it